applications/plugins/LMConstraint/examples/LMConstraintCollisionResponse.scn

```diff
@@ -4,6 +4,9 @@ This scene involves collision between rigid and deformable ojects with different
 and stiffness coefficients for the collision models. 
 -->
 <Node >
+    <RequiredPlugin name="LMConstraint"/>
+    <RequiredPlugin name="SofaDistanceGrid"/>
```

Comment:
I'm really not sure about that. 
Isn't it weird to expose the fact that SofaDistanceGrid is *internally* used by LMConstriant ?